qctrlopencontrols/dynamic_decoupling_sequences/dynamic_decoupling_sequence.py

```diff
@@ -39,29 +39,28 @@
 
 
 class DynamicDecouplingSequence:
-    """
+    r"""
     Create a dynamic decoupling sequence.
```

Comment:
```suggestion
    Creates a dynamic decoupling sequence.
```

I don't think we have any standards in place for this, but it seems all the other docstrings start with "Creates", so I suppose we might as well start this one in the same way

Reply:
Agree. `Creates` seems to be the pattern we are following.